src/api/app/views/webui2/webui/request/_show_overview.html.haml

```diff
@@ -0,0 +1,13 @@
+%h3
+  %p
+    = pagetitle
+    = "(#{state})"
+#description-text
+  - if description.blank?
+    %i No description set
+  - else
+    = description_wrapper(toggle_sliced_text(description))
```

Comment:
We don't want to use `description_wrapper` in the webui2 views. On top of this, the `if/else` in this view is almost exactly what `description_wrapper` does.

```
  # TODO: bento_only
  def description_wrapper(description)
    if description.blank?
      content_tag(:p, id: 'description-text') do
        content_tag(:i, 'No description set')
      end
    else
      content_tag(:pre, description, id: 'description-text', class: 'plain')
    end
  end
```

We should do it directly in the view.

Reply:
I don't know who are "We" here, but, there is some javascript going on here. To use collapse do not work, because we want to show some text and then truncate it with the "+" to expand it. Collapse will break it.

Because requests use the changes as description, they are normally really long, so the way done in the project#show or package#show, IMO do not fit here. I would prefer to have this change done later,  in this Deliverable,  if we have some free cycles in the end to do it...

Reply:
'We' use https://getbootstrap.com/docs/4.1/components/collapse/ everywhere else